eng/common/scripts/Verify-Links.ps1

```diff
@@ -18,13 +18,18 @@ param (
   # the substitute branch name or SHA commit
   [string] $branchReplacementName = "",
   # flag to allow checking against azure sdk link guidance. Check link guidance here: https://aka.ms/azsdk/guideline/links
-  [bool] $checkLinkGuidance = $false
+  [bool] $checkLinkGuidance = $false,
+  # UserAgent to be configured for web request. Default to PSUserAgent. 
+  [string] $userAgent = "Microsoft.PowerShell.Commands.PSUserAgent"
```

Comment:
I wonder if we should default it to the most common browser useragent string so we can seem like a browser when checking.

Reply:
We can choose either IE or Chrome here. I can set to Chrome if no other preference.

Reply:
The chrome user agent string would be fine.